.github/workflows/format.yml

```diff
@@ -10,10 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Toolchain setup
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
```

Comment:
Should this be `@nightly` instead of `@master`?

Reply:
Not needed since the tool chain is specified below. Did it that way since a component is specified.